src/Adapter/Product/Combination/Repository/CombinationRepository.php

```diff
@@ -234,6 +234,31 @@ public function getProductId(CombinationId $combinationId): ProductId
         return new ProductId((int) $result['id_product']);
     }
 
+    /**
+     * @param CombinationId $combinationId
+     *
+     * @return string
+     */
+    private function getCombinationReference(CombinationId $combinationId): string
+    {
+        $qb = $this->connection->createQueryBuilder();
+        $qb
+            ->select('pa.reference')
+            ->from($this->dbPrefix . 'product_attribute', 'pa')
+            ->andWhere('pa.id_product_attribute = :combinationId')
+            ->setParameter('combinationId', $combinationId->getValue())
+        ;
+
+        $result = $qb->execute()->fetchAssociative();
+        $reference = '';
+
+        if (!empty($result['reference'])) {
+            $reference = $result['reference'];
+        }
+
+        return $reference;
```

Comment:
```suggestion
        $result = $qb->execute()->fetchAssociative();

        return $result['reference'] ?? '';
```

I think this will be enough :)